fw/tls.c

```diff
@@ -40,14 +40,12 @@
 /**
  * Global level TLS configuration.
  *
- * @cfg			- common tls configuration for all vhosts;
- * @allow_any_sni	- If set, all the unknown SNI are matched to default
  *			  vhost.
  */
-static struct {
-	TlsCfg		cfg;
-	bool		allow_any_sni;
-} tfw_tls;
+/** Common tls configuration for all vhosts; */
+static TlsCfg		tfw_tls_cfg;
+/** If set, all the unknown SNI are matched to default vhost. */
+bool		    tfw_tls_allow_any_sni;
```

Comment:
It's about style and up to you, but maybe we could introduce `getter` for `tfw_tls.allow_any_sni` and don't "globalize" the `allow_any_sni` var?

Reply:
I thought getters are not an accepted practice on project. If you don't mind, I'd keep things as it is.